wiki/Gameplay/Input_device/Beatmania_controller/en.md

```diff
@@ -0,0 +1,11 @@
+# Beatmania Controller
+
+A **Beatmania controller** is a type of game controller that is compatible with the *[Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")* series of games. 
+Due to their nature, Beatmania controllers are most often used as input devices in [osu!mania](/wiki/Game_Modes/osu!mania).
+
+## Input methods
+
+For players who have a Playstation 2 (PS2) *Beatmania* controller, they can use an adaptor to connect the controller to their computer via USB and bind the keys to their desired keys. 
+For players who have a *DJ DAO IIDX* controller, they can plug in the controller to their computer via USB and bind the keys in the options menu.
```

Comment:
Remarkjs will have a fit with these lines. Either make into a "list" or add another sentence or two to make them separate paragraphs.